src/amo/pages/Block/index.js

```diff
@@ -178,12 +178,16 @@ export class BlockBase extends React.Component<InternalProps> {
             />
             <p className="Block-metadata">
               {block ? (
-                i18n.sprintf(
-                  i18n.gettext('Versions blocked: %(min)s to %(max)s.'),
-                  {
-                    min: block.min_version,
-                    max: block.max_version,
-                  },
+                block.min_version === '0' && block.max_version === '*' ? (
+                  i18n.sprintf(
+                    i18n.gettext('Versions blocked: %(min)s to %(max)s.'),
+                    {
+                      min: block.min_version,
+                      max: block.max_version,
+                    },
+                  )
+                ) : (
+                  i18n.gettext('Versions blocked: all versions.')
```

Comment:
Shouldn't it be the other way around?

Write a test case in `TestBlock.js` to make sure it works as intended.

Reply:
Oh sorry! Yes I will update the pr.

Reply:
Yes I will write the test.